### PR TITLE
CSS Tools Slides Update

### PR DESCRIPTION
**Pull request by trucnguyen**
@brendajin Could you take a look when you get a chance?
CC @mjawili 

- Adding more image based examples
- Updating slides with modern examples (e.g. YUI is no longer maintained, Foundation js example instead)

### CODE REVIEW

File: csstools/css/normalize.css [hunk 1]
```diff
@@ -0,0 +1,504 @@
+/*! normalize.css 2012-03-11T12:53 UTC - http://github.com/necolas/normalize.css */
```
Comment (brendajin) [inline csstools/css/normalize.css]: This file exists under `common/`. Might be good to refer to that one.

File: csstools/css/presentation.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
+body {
```
Comment (brendajin) [inline csstools/css/presentation.css]: same thing for this file

File: csstools/js/fathom.js [hunk 3]
```diff
@@ -0,0 +1,388 @@
+/*
```
Comment (brendajin) [inline csstools/js/fathom.js]: also in `common/`

File: csstools/index.html [hunk 4]
```diff
@@ -777,12 +514,9 @@
     </div>
 
 
-
-
-
     <div class="slide normal">
      <br><br>
-     <header><h1><a href="exercise_bootstrap_descrip.html">Exercise Time!</a></h1></header>
+     <header><h1><a href="exercises/exercise_bootstrap_descrip.html">Exercise Time!</a></h1></header>
```
Comment (brendajin) [inline csstools/index.html]: thanks for reorganizing these

**Comment by trucnguyen** [comment]
@brendajin Updated the linked css and js to point to ../common (common should have those files under in subdir js/ and css/ within it but that's another story or PR)

**Pull request merged**